sdk/storage/Azure.Storage.Blobs.DataMovement/src/BlobScanner.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Specialized;
+
+namespace Azure.Storage.Blobs.DataMovement
+{
+    /// <summary>
+    /// AzureContainerScanner class.
```

Comment:
If you generate docs for the sake of analyzer please just leave "TODO" as a content.